### PR TITLE
Enhancement texture path search rule

### PR DESCRIPTION
**Pull request by stefaniapedrazzi**
Related to https://github.com/omichel/webots/pull/6216

### CODE REVIEW

**Comment by stefaniapedrazzi** [comment]
:+1:

**Comment by tn12787** [comment]
Semi-related to your changes, "is_existing" is not nice at all. It should be "exists". It might be nice to fix that here.

**Comment by stefaniapedrazzi** [comment]
:+1:

File: reference/imagetexture.md [hunk 1]
```diff
@@ -91,11 +91,13 @@ if is_absolute(url[i]) then
...
 else
-  if defined_in_a_PROTO(current_node) and is_existing(PROTO_path + url[i])
+  if defined_in_a_project_PROTO(current_node) and exists(PROTO_path + url[i])
     return PROTO_path + url[i]
-  else if is_existing(world_path + url[i])
+  else if exists(world_path + url[i])
     return world_path + url[i]
-  else if is_existing(generic_textures_path + url[i])
+  else if defined_in_a_webots_PROTO(current_node) and exists(PROTO_path + url[i])
+      return PROTO_path + url[i]
```
Comment (omichel) [inline reference/imagetexture.md]: Indentation looks odd here. There seems to be extra spaces.

**Comment by stefaniapedrazzi** [comment]
:+1:

**Pull request merged**